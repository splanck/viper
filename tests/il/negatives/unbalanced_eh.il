il 0.1.2
func @main() -> void {
entry:
  eh.push ^handler
  ret
handler ^handler(%err:Error, %tok:ResumeTok):
  eh.entry
  resume.next %tok
}
