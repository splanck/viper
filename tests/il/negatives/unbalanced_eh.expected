error: verify.eh.unreleased: main:entry: ret: unmatched eh.push depth 1; path: entry
